index.js

```diff
@@ -0,0 +1,71 @@
+var _ = require('lodash');
+
+module.exports = {
+  moogBundle: {
+    modules: ['apostrophe-svg-sprites-widgets'],
+    directory: 'lib/modules'
+  },
+
+  name: 'apostrophe-svg-sprites',
+  extend: 'apostrophe-images',
```

Comment:
Are we reusing anything specific by extending apostrophe-images and not apostrophe-pieces?

Reply:
subclassing pieces now